index.js

```diff
@@ -82,7 +81,9 @@ module.exports = function(opt) {
 		file.contents = mangled.code;
 
 		if (file.sourceMap) {
-			applySourceMap(file, mangled.map);
+			var sourceMap = JSON.parse(mangled.map);
+			sourceMap.sources = [file.relative];
```

Comment:
@terinjokes Yes, it does. The sources array set here is just for the source map of this transformation. It is merged with the source map of the incoming file in `applySourceMap`. The incoming source map can have an array of multiple sources.